audio/osc2midi/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+
+github.setup            ssj71 OSC2MIDI 0.2.3
+master_sites            https://github.com/ssj71/OSC2MIDI/archive/
+distname                v${version}
```

Comment:
Would putting a "v" at the end of github.setup be better style?

Reply:
It didn't work, when I tried it.

If there's a secret to this I'm not getting, I'd love to hear it.

Reply:
If you look at https://github.com/ssj71/OSC2MIDI/tags you can see that the latest tag does begin with the prefix "v", so the correct thing to do is to specify "v" as the tag prefix, in the 4th argument of `github.setup`. Do not override `distname` or `master_sites`; the portgroup sets them correctly for you.